bundles/org.openhab.binding.deconz/README.md

```diff
@@ -235,6 +239,8 @@ Bridge deconz:deconz:homeserver [ host="192.168.0.10", apikey="ABCDEFGHIJ" ] {
 Switch                  Livingroom_Presence     "Presence Livingroom [%s]"          <motion>        { channel="deconz:presencesensor:homeserver:livingroom-presence:presence" }
 Number:Temperature      Livingroom_Temperature  "Temperature Livingroom [%.1f °C]"  <temperature>   { channel="deconz:temperaturesensor:homeserver:livingroom-temperature:temperature" }
 Number:Dimensionless    Livingroom_Humidity     "Humidity Livingroom [%.1f %%]"     <humidity>      { channel="deconz:humiditysensor:homeserver:livingroom-humidity:humidity" }
+String                  LIVING_ROOM_VOC_LABEL    "Air quality Livingroom [%s]"      <none>          { channel="deconz:airqualitysensor:homeserver:livingroom-voc:airquality" }
+Number                  LIVING_ROOM_VOC          "Air quality [%d ppb]"             <none>          {           channel="deconz:airqualitysensor:homeserver:livingroom-voc:airqualityppb" }
```

Comment:
This is now `Number:Dimensionless`.

Reply:
Doc is adapted.

Reply:
Seems like it was not pushed?

Reply:
With Commit https://github.com/openhab/openhab-addons/pull/11885/commits/75b1cb7075e39b2178c2e76f8a2b2364a78fff18 it was pushed.

Reply:
Hmm, my comment was for **airqualityppb** being documented as `Number`, when it's now `Number:Dimensionless`. I don't see this change in current version or in that commit.

Reply:
Now I understood it and changed it.